mobile/test/common/integration/client_integration_test.cc

```diff
@@ -131,12 +131,25 @@ void ClientIntegrationTest::trickleTest() {
   terminal_callback_.waitReady();
 }
 
-TEST_P(ClientIntegrationTest, Trickle) {
+TEST_P(ClientIntegrationTest, TrickleNoMinChunk) {
+  min_chunk_size_ = 0;
   trickleTest();
   ASSERT_LE(cc_.on_data_calls, 11);
 }
 
-TEST_P(ClientIntegrationTest, TrickleExplicitFlowControl) {
+TEST_P(ClientIntegrationTest, TrickleNoNoMinChunkExplicitFlowControl) {
+  min_chunk_size_ = 0;
+  explicit_flow_control_ = true;
+  trickleTest();
+  ASSERT_LE(cc_.on_data_calls, 11);
+}
+
+TEST_P(ClientIntegrationTest, TrickleMinChunk) {
+  trickleTest();
```

Comment:
I thought this test would've first set min_chunk_size_ to something bigger, than make sure on_data_calls is some other number (less than 11 I presume)?